src/registrar/models/nameserver.py

```diff
@@ -0,0 +1,14 @@
+from .host import Host
+
+
+class Nameserver(Host):
+    """
+    A nameserver is a host which has been delegated to respond to DNS queries.
+
+    The registry is the source of truth for this data.
+
+    This model exists ONLY to allow a new registrant to draft DNS entries
+    before their application is approved.
```

Comment:
**suggestion (non-blocking)**: It isn't always obvious to me in Django models that use class inheritance that I should look at the super class to learn about the fields that are allowed here. Could we have a comment to that effect, maybe above  the `pass` that says "there is nothing here because all of the fields are defined over there on the Host class"?

Reply:
Done.